<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-OFI38GDM" failures="1" tests="1" name="nopCommerce1 Test2" time="24.284" errors="0" timestamp="2021-12-26T16:43:39 IST">
  <testcase classname="com.nopCommerce1.TestCases.TC_LoginTest_001" name="loginTest" time="19.588">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopCommerce1.TestCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- nopCommerce1 Test2 -->
